src/components/SearchBox.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+class SearchBox extends React.PureComponent {
```

Comment:
1. Make an example where you can see the benefit of making this component pure.
2. Class is missing a constructor, It might backfire in some cases.
https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class